camelot/model/party.py

```diff
@@ -170,18 +170,18 @@ class GeographicBoundaryAlternativeName(Entity):
     __tablename__ = 'geographic_boundary_alternative_name'
     
     name = schema.Column(Unicode(100), nullable=False)
-    row_type = schema.Column(sqlalchemy.types.Unicode(40), nullable=True, index=True)
+    row_type = schema.Column(sqlalchemy.types.Unicode(40), nullable=False, index=True)
     language = schema.Column(sqlalchemy.types.Unicode(6), nullable=True)
     
     alternative_name_for_id = schema.Column(sqlalchemy.types.Integer(),
                                             schema.ForeignKey(GeographicBoundary.id, ondelete='cascade', onupdate='cascade'),
                                             nullable = False,
                                             index = True)
-    alternative_name_for = orm.relationship(GeographicBoundary, backref=orm.backref('alternative_names', cascade='all, delete, delete-orphan'))
+    alternative_name_for = orm.relationship(GeographicBoundary, backref=orm.backref('alternative_names', cascade='all, delete, delete-orphan', overlaps="translations"))
     
     __mapper_args__ = {
         'polymorphic_on' : row_type,
-        'polymorphic_identity': None,
```

Comment:
Een None polymorphic identity werkte vroeger blijkbaar per ongeluk, in 1.4. niet meer.
https://github.com/sqlalchemy/sqlalchemy/discussions/6642